assets/scss/_nav.scss

```diff
@@ -41,9 +41,8 @@
         text-transform: none;
         text-align: middle;
 
-        .nav-link {
-            display: inline-block;
-            margin-right: -30px;
```

Comment:
We don't have `.nav-link`s inside `.navbar-brand` and never did as far back as I could search, so I'm deleting this styel